pkg/service/service.go

```diff
@@ -61,6 +62,15 @@ func CreateService(client *occlient.Client, serviceName string, serviceType stri
 	return nil
 }
 
+// CreateOperatorService creates new service (actually a Deployment) from OperatorHub
+func CreateOperatorService(client *kclient.Client, serviceName string, serviceType string, crd string, parameters map[string]string, applicationName, group, version, resource string, exampleCR map[string]interface{}) error {
```

Comment:
too many args on this function

Reply:
and it seems you are not using `serviceName`, `serviceType` , `parameters`  and `applicationName`